src/bundle/Resources/translations/custom_tags.en.xliff

```diff
@@ -0,0 +1,131 @@
+      <note>The source node in most cases contains the sample message as written by the developer. If it looks like a dot-delimitted string such as "form.label.firstname", then the developer has not provided a default message.</note>
+    </header>
+    <body>
+      <trans-unit id="7ddd17c1a96742c10988612627d3bf5134bcd912" resname="ezrichtext.custom_tags.ezfacebook.attributes.post_url.label">
+        <source>Post URL</source>
+        <target>Post URL</target>
+        <note>key: ezrichtext.custom_tags.ezfacebook.attributes.post_url.label</note>
+      </trans-unit>
+      <trans-unit id="14df6986afb2121f6d8f3a8e3fd3cc2e8719f28e" resname="ezrichtext.custom_tags.ezfacebook.attributes.width.label">
+        <source>Width</source>
+        <target>Width</target>
+        <note>key: ezrichtext.custom_tags.ezfacebook.attributes.width.label</note>
+      </trans-unit>
+      <trans-unit id="04a09b59ee06e7996670b684a9ca256d792992d2" resname="ezrichtext.custom_tags.ezfacebook.description">
+        <source>Embed Facebook post</source>
+        <target>Embed Facebook post</target>
+        <note>key: ezrichtext.custom_tags.ezfacebook.description</note>
+      </trans-unit>
+      <trans-unit id="48fc0ce4ff40934d1fff06a56055c9ad73f634f4" resname="ezrichtext.custom_tags.ezfacebook.label">
+        <source>Facebook</source>
+        <target>Facebook</target>
+        <note>key: ezrichtext.custom_tags.ezfacebook.label</note>
+      </trans-unit>
+      <trans-unit id="a711cbe9c11ea82cc8ad02c93af5727dceb8b34d" resname="ezrichtext.custom_tags.eztwitter.attributes.cards.choice.hidden.label">
+        <source>Hidden</source>
+        <target>Hidden</target>
+        <note>key: ezrichtext.custom_tags.eztwitter.attributes.cards.choice.hidden.label</note>
+      </trans-unit>
+      <trans-unit id="b7ef5c07f5cbc982f23e547bc1e1326752853018" resname="ezrichtext.custom_tags.eztwitter.attributes.cards.label">
+        <source>Cards</source>
+        <target>Cards</target>
+        <note>key: ezrichtext.custom_tags.eztwitter.attributes.cards.label</note>
+      </trans-unit>
+      <trans-unit id="333d02160158768833ecf1aa1bde5b60a9c3f7d1" resname="ezrichtext.custom_tags.eztwitter.attributes.conversation.choice.none.label">
+        <source>None</source>
+        <target>None</target>
+        <note>key: ezrichtext.custom_tags.eztwitter.attributes.conversation.choice.none.label</note>
+      </trans-unit>
+      <trans-unit id="5671ef11994be024b75d6360144708538714d952" resname="ezrichtext.custom_tags.eztwitter.attributes.conversation.label">
+        <source>Conversation</source>
+        <target>Conversation</target>
+        <note>key: ezrichtext.custom_tags.eztwitter.attributes.conversation.label</note>
+      </trans-unit>
+      <trans-unit id="7fa382eb492f241c0377d64e8ad5c8ab8aa740fb" resname="ezrichtext.custom_tags.eztwitter.attributes.dnt.label">
+        <source>Do not target</source>
+        <target>Do not target</target>
```

Comment:
I would change to Don't target

Reply:
Fixed in https://github.com/ibexa/fieldtype-richtext/pull/162/commits/458e5fff13cfcd2127613d8c72ff891b59480cae